.github/actions/build-electron/action.yml

```diff
@@ -71,18 +71,22 @@ runs:
         cd src
         e build --target electron:electron_dist_zip -j $NUMBER_OF_NINJA_PROCESSES
         if [ "${{ inputs.is-asan }}" != "true" ]; then
-          target_os=${{ inputs.target-platform == 'linux' && 'linux' || 'mac'}}
+          target_os=${{ inputs.target-platform == 'linux' && 'linux' || (inputs.target-platform == 'windows' && 'win' || 'mac') }}
```

Comment:
is there a specific reason we're using 'windows' some places and 'win' others? it's a little confusing 🤔

Reply:
+1